rllib/env/vector/vector_multi_agent_env.py

```diff
@@ -0,0 +1,83 @@
+import gymnasium as gym
+import numpy as np
+
+from gymnasium.core import RenderFrame
+from gymnasium.envs.registration import EnvSpec
+from gymnasium.utils import seeding
+from typing import Any, Dict, Optional, Tuple, TypeVar
+
+
+ArrayType = TypeVar("ArrayType")
+
+
+class VectorMultiAgentEnv:
```

Comment:
Does this API match the single-agent VectorEnv API?

Are there any important diffs?

Reply:
It matches single-agent VectorEnv but with the difference of the spaces